src/rendering/batcher/shared/Batcher.ts

```diff
@@ -240,7 +240,7 @@ export class Batcher
         let start = this._batchIndexStart;
 
         let action: BatchAction = 'startBatch';
-        let batch = this._batchPool[this._batchPoolIndex++] || new Batch();
+        let batch = this._batchPool[this._batchPoolIndex++] ||= new Batch();
```

Comment:
I think we are missing a `.bindGroup = null` in `_finishBatch`. I don't see it being reset anywhere. Not sure if it matters.